### PR TITLE
RDKTV-13125: WPEFramework timeout during shutdown

### PR DESCRIPTION
**Pull request by livinsunny519**
Reason for change:Due to maintenance running the task seems to be
blocked.
Test Procedure:Refer Ticket.
Risks: Low

### CODE REVIEW

**Comment by livinsunny519** [comment]
Developer Testing in progress. Do not merge.

File: MaintenanceManager/MaintenanceManager.cpp [hunk 1]
```diff
@@ -237,16 +237,16 @@ namespace WPEFramework {
...
+                    LOGINFO("DBG After Lock");
                     cmd=task_names_foreground[i].c_str();
                     cmd+=" &";
                     cmd+="\0";
                     m_task_map[task_names_foreground[i].c_str()]=true;
                     LOGINFO("Starting Script (USM) :  %s \n", cmd.c_str());
-                    if (!m_abort_flag){
-                        system(cmd.c_str());
-                    }
+                    system(cmd.c_str());
```
Comment (npoltorapavlo) [inline MaintenanceManager/MaintenanceManager.cpp:249]: is it expected that the script will start when m_abort_flag is true?

Comment (livinsunny519) [reply MaintenanceManager/MaintenanceManager.cpp]: I have modified it. Please check

Comment (npoltorapavlo) [reply MaintenanceManager/MaintenanceManager.cpp]: looks good

**Comment by livinsunny519** [comment]
Created https://github.com/rdkcentral/rdkservices/pull/2077

**Pull request closed without merging**